src/App.js

```diff
@@ -3,10 +3,17 @@ import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 import CommonLayout from "./components/CommonLayout";
 import "./App.css";
 
+import CohortApplication from './pages/CohortApplication';
+
 function App() {
   return (
     <Router>
       <Switch>
+        <Route exact path="/cohort-application">
```

Comment:
for now its fine but after it get merged with master iit should go under admin

Reply:
Ok we can discuss this tomorrow.